OPERATOR.rst

```diff
@@ -448,6 +448,52 @@ not, then the PR's original author should.
 Once the base branch is restored, the ``Reopen PR`` button should again be
 clickable on the chained PR.
 
+Integration test time out
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A warning sign that IT will likely time out is when Azul starts dividing up
+bundle partitions. Run the BigQuery query below against the indexer log groups
+to see if this is the case::
+
+    fields @timestamp, @message
+    | filter @message like 'Dividing partition'
+    | sort @timestamp desc
+    | limit 20
+
```

Comment:
It feels to me like this is a redundant step, as it is now it implicitly implies any `Dividing partition` messages are a warning sign, is that the case?